scope/src/debug.c

```diff
@@ -347,7 +348,7 @@ static void gdb_finalize(void)
 	if (send_channel)
 	{
 		g_io_channel_shutdown(send_channel, FALSE, NULL);
-		g_io_channel_unref(send_channel);
+		g_io_channel_unref(send_channel); // cppcheck-suppress doubleFree // should be fine
```

Comment:
I have no idea if this correct, maybe the unref call is indeed too much?

Reply:
AFAICT the unref is needed, the `g_io_channel_shutdown()` documentation, erm ...source ;-) shows it doesn't unref it.